lib/IO/Socket/SSL.pm6

```diff
@@ -126,6 +125,10 @@ method print(Str $s) {
     $!ssl.write($s);
 }
 
+method put(Str $s) {
+    $!ssl.write($s ~ "\n");
```

Comment:
Why not use `$.nl-out`?

Reply:
Well spotted. I changed it to `$!nl-out`, or do you think it's preferable to use the accessor?

Reply:
I generally try to avoid calling private accessors that come from outside the class (composed in from roles). It is fine (to me) for `IO::Socket` to call `$!nl-out` because it is declared in that class and thus the implementation details can be hidden. Using `$.nl-out` outside of `IO::Socket` is ok because it ensures it will use the public interface (maybe in the future there is no `$!nl-out`, only `method nl-out { ... }`)

Reply:
Good point. Amended